src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -11,15 +12,35 @@
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
+    public static final int PAGE_CONTAIN = 3;
```

Comment:
Good to convert magic number to constants. However, the name can be more intuitive. E.g. MAX_PERSON_PER_PAGE or etc.